src/Bicep.LangServer.IntegrationTests/Completions/SnippetTemplates/res-template-spec/main.bicep

```diff
@@ -0,0 +1,5 @@
+﻿// $1 = 'myTemplateSpecName'
+// $2 = 'myTemplateSpecFriendlyName'
+// $3 = 'myTemplateSpecDisplayName'
```

Comment:
We've been trying to stick to the convention of same name as the identifier for the resource name, and the name of the property for the other placeholder values. 

e.g.
// $1 = 'templateSpec'
// $2 = 'desciption'
// $3 = 'displayName'

Reply:
Make sense! Updated accordingly